### ISSUE TITLE
aql doesn't escape double-quotes when outputting as JSON

### ISSUE THREAD

**Issue by kennethjor** [aerospike/aerospike-server#73]
To reproduce:

1. Execute the following aql statement: ` INSERT INTO test.escape (PK, data) VALUES ('a','{"key":"val"}')`
2. Execute from command line: `aql -o json -c "select * from test.escape"`

**Result**

```
[
  {
    "data": "{"key":"val"}"
  }
]
```

**Expected:**

```
[
  {
    "data": "{\"key\":\"val\"}"
  }
]
```

Note the escaped double quotes. This results in invalid JSON being outputted.

**Comment by wchu-citrusleaf** [comment]
Thanks for the bug report.

**Comment by kennethjor** [comment]
Hi guys, is there any progress on this?

**Comment by wchu-citrusleaf** [comment]
In testing stage. Likely be out with next release.

**Comment by kennethjor** [comment]
Thank you very much, looking forward to it :)

**Comment by wchu-citrusleaf** [comment]
Released in 3.6.0
AQL) [AER-3734] - [Github#73] Fix double quote escape: Changed json output to use libjansson so all other escapes should be correct too.
